containers/datalab/Dockerfile.in

```diff
@@ -50,7 +50,7 @@ RUN pip install -U tornado==4.3 \
                    argparse==1.2.1 \
                    mock==1.2.0 \
                    requests==2.9.1 \
-                   oauth2client==1.5.2 \
+                   oauth2client==2.0.2 \
```

Comment:
The latest version of dataflow is no longer pinned to oauth2client v1.5.2:
https://github.com/GoogleCloudPlatform/DataflowPythonSDK/releases/tag/v0.2.3

Does this help with removing the dependency within Datalab?

Also, the TensorFlow sample notebook executes fine with the updated oauth client.